website/docs/docs/build/metricflow-cli.md

```diff
@@ -0,0 +1,247 @@
+---
+title: MetricFlow CLI
+id: metricflow-cli
+description: "Query metrics and metadata in your dbt project with the metricflow cli"
+sidebar_label: "CLI"
+tags: [Metrics, Semantic Layer]
+---
+
+After you have defined metrics in your dbt project, you can easily query metrics, dimensions, dimension values, and validate your configs using the MetricFlow CLI.
+
+## Installation
+1. Create or activate your virtual environment.`python -m venv venv`
+2. Run `pip install dbt-metricflow`
```

Comment:
@Jstein77 is it `pip install dbt-metricflow` or `pip install metricflow` per the readme? 

I'm on python3.11/site-packages (23.1.2), so when i tried installing using `dbt-metricflow`, i got the below error:

```(mwong-metrics) ➜  ~ git:(current) ✗ pip install dbt-metricflow
ERROR: Ignored the following versions that require a different python version: 0.0.1 Requires-Python <3.10,>=3.8; 0.0.2 Requires-Python <3.10,>=3.8; 0.0.3 Requires-Python <3.10,>=3.8
ERROR: Could not find a version that satisfies the requirement dbt-metricflow (from versions: none)
ERROR: No matching distribution found for dbt-metricflow```

but when i used `pip install metricflow` as specified in the cli package github readme, it installed successfully. 

<img width="552" alt="Screenshot 2023-07-19 at 10 08 54" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/571885f9-b50a-4257-b173-cd1b730b43a5">

Reply:
I think this is a bug. The preferred way to install the cli is `pip install dbt-metricflow`

Reply:
https://github.com/dbt-labs/metricflow/pull/663 should be resolved when this is merged